pubtools/pulplib/_impl/model/repository/yum.py

```diff
@@ -1,9 +1,86 @@
-from .base import Repository, repo_type
+from .base import Repository, SyncOptions, repo_type
 from ..frozenlist import FrozenList
 from ..attr import pulp_attrib
 from ... import compat_attr as attr
 
 
+@attr.s(kw_only=True, frozen=True)
+class YumSyncOptions(SyncOptions):
+    """Options controlling a container repository
+    :meth:`~pubtools.pulplib.YumRepository.sync`.
+    """
+
+    query_auth_token = pulp_attrib(default=None, type=str)
+    """An authorization token that will be added to every request made to the feed URL's server
+    """
+
+    validate = pulp_attrib(default=None, type=bool)
```

Comment:
Same comment as with the FileSyncOptions, can we leave it out please?  Setting validate=False seems useless.